static/app/views/dashboardsV2/manage/index.tsx

```diff
@@ -251,4 +312,27 @@ const StyledActions = styled('div')`
   }
 `;
 
+const SwitchContainer = styled('div')`
+  font-size: ${p => p.theme.fontSizeLarge};
+  display: inline;
+  padding-right: ${space(2)};
+`;
+
+const TemplateSwitch = styled(Switch)`
+  vertical-align: middle;
+  display: inline;
+  margin-left: ${space(1)};
+`;
+
+const ButtonContainer = styled('div')`
+  display: inline;
+`;
+
+const TemplateContainer = styled('div')`
```

Comment:
We should update the css here to add a breakpoint to the cards stack after a certain window size otherwise it overflows and gets cut off.

![Screen Shot 2021-12-15 at 9 25 47 AM](https://user-images.githubusercontent.com/63818634/146204486-ee7e729f-4fa6-4eaa-acaf-dbb80b012724.png)

Something like this (untested)
```
@media (max-width: ${p => p.theme.breakpoints[0]}) {
    grid-template-columns: repeat(2, 1fr);
  }
```